webpages/register.html

```diff
@@ -0,0 +1,139 @@
+<!doctype html>
```

Comment:
when this page contains extra guff such as t&c. This page could react to you already being registered and only allow you to change your user name